src/sentry/api/endpoints/admin_project_configs.py

```diff
@@ -0,0 +1,42 @@
+from django.http import Http404
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.base import Endpoint, pending_silo_endpoint
+from sentry.api.permissions import SuperuserPermission
+from sentry.models import Project
+from sentry.relay import projectconfig_cache
+
+
+@pending_silo_endpoint
+class AdminRelayProjectConfigsEndpoint(Endpoint):
+    permission_classes = (SuperuserPermission,)
+
+    def get(self, request: Request) -> Response:
+        project_id = request.GET.get("project-id")
```

Comment:
Most of our query/request data keys use `camelBack` style casing. There isn't much `kebab-case` in API endpoints.